routes/index.js

```diff
@@ -19,6 +19,7 @@ app.use("/users/status", require("./userStatus.js"));
 app.use("/users", require("./users.js"));
 app.use("/profileDiffs", require("./profileDiffs.js"));
 app.use("/wallet", require("./wallets.js"));
+app.use("/migrations", require("./userMigrations.js"));
```

Comment:
if you are aware @ankushdharkar has strongly advised against approving pull requests that don't adhere to the correct naming convention. As `migrations` isn't a resource, could you kindly consider relocating this route under `\users` ?